malcolm/modules/ADCore/parts/detectordriverpart.py

```diff
@@ -23,11 +20,19 @@
 with Anno("Directory to write data to"):
     AFileDir = str
 
+# Pull re-used annotypes into our namespace in case we are subclassed
+APartName = APartName
+AMri = builtin.parts.AMri
+AbortHook = scanning.hooks.AbortHook
```

Comment:
I think we shouldn't pull hook annotypes into our namespace, just __init__ types